lib/carelink/csv/minimed.com.js

```diff
@@ -0,0 +1,18 @@
+var parsing = require('../parsing.js');
+
+var RV = 'Raw-Values';
+
+module.exports = {
+  pumpTypesToRead: {
+    BGCapturedOnPump: {
+      RAW_TYPE: 'BGCapturedOnPump',
+      RAW_VALUES: {
+        AMOUNT: parsing.asNumber([RV, 'AMOUNT']),
+        ACTION_REQUESTOR: parsing.extract([RV, 'ACTION_REQUESTOR'])
+      }
+    }
+  },
+  cgmTypesToRead: {
```

Comment:
Separating out pump types from CGM types is something that I had to do in the UTC bootstrapping work, so this is just looking forward.